src/Hypergrid.js

```diff
@@ -1286,6 +1280,19 @@ Hypergrid.prototype = {
 
     /**
      * @memberOf Hypergrid.prototype
+     * @summary Shut down the current cell editor without saving the edited value.
+     * @returns {boolean} `true` if we were editing; `false` if we were not.
+     */
+    cancelEditing: function() {
+        var wasEditing = !!this.cellEditor;
+        if (wasEditing) {
+            this.cellEditor.cancelEditing();
```

Comment:
Yes. I thought we could come up with a minimal list, in conference. Let's do tomorrow in person.